doc/langref.html.in

```diff
@@ -7824,7 +7831,7 @@ comptime {
       {#header_open|@atomicLoad#}
       <pre>{#syntax#}@atomicLoad(comptime T: type, ptr: *const T, comptime ordering: builtin.AtomicOrder) T{#endsyntax#}</pre>
       <p>
-      This builtin function atomically dereferences a pointer and returns the value.
+      This builtin function dereferences a pointer to a {#syntax#}T{#endsyntax#} and atomically returns the value.
```

Comment:
I was trying to avoid implying that the reading of the pointer address is not atomic.  But as you say, "dereference" covers both the address reading/resolve and the fetch.  I fear trying to be too precise here will just add a lot of verbiage, so I think I'll just put "atomically" back where it was for now and back away slowly.  (I glanced through GCC, Java, Nim and Erlang docs, and they didn't have any obviously better wording for this operation.)